pkg/daemon/daemon.go

```diff
@@ -107,8 +115,16 @@ func NewDaemon() (Daemon, error) {
 		return nil, err
 	}
 
-	if err := smClient.Validate(); err != nil {
-		return nil, err
+	// Try to validate if subnet manager is reachable in backoff loop
+	var validateErr error
```

Comment:
i think for the initial startup of the daemon, we don't need to do exponential backoff WDYT ?

Reply:
Not sure. What if daemon initialized before manager was configured/becomes reachable?

Reply:
OK lets add warning to the log in case of failure when retrying like other cases